python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -0,0 +1,1219 @@
+        default=True,
+    )
+
+    # Variance construction.
+    doVariance = pexConfig.Field(
+        dtype=bool,
+        doc="Calculate variance?",
+        default=True
+    )
+    gain = pexConfig.Field(
+        dtype=float,
+        doc="The gain to use if no Detector is present in the Exposure (ignored if NaN).",
+        default=float("NaN"),
+    )
+    usePtcGains = pexConfig.Field(
+        dtype=bool,
+        doc="Use the gain values from the Photon Transfer Curve?",
+        default=False,
+    )
+    readNoise = pexConfig.Field(
+        dtype=float,
+        doc="The read noise to use if no Detector is present in the Exposure.",
+        default=0.0,
+    )
+    doEmpiricalReadNoise = pexConfig.Field(
+        dtype=bool,
+        doc="Calculate empirical read noise instead of value from AmpInfo data?",
+        default=False,
+    )
+    usePtcReadNoise = pexConfig.Field(
+        dtype=bool,
+        doc="Use readnoise values from the Photon Transfer Curve?",
+        default=False,
+    )
+    maskNegativeVariance = pexConfig.Field(
+        dtype=bool,
+        doc="Mask pixels that claim a negative variance.  This likely indicates a failure "
+        "in the measurement of the overscan at an edge due to the data falling off faster "
+        "than the overscan model can account for it.",
+        default=True,
+    )
+    negativeVarianceMaskName = pexConfig.Field(
+        dtype=str,
+        doc="Mask plane to use to mark pixels with negative variance, if `maskNegativeVariance` is True.",
+        default="BAD",
+    )
+    saturatedMaskName = pexConfig.Field(
+        dtype=str,
+        doc="Name of mask plane to use in saturation detection and interpolation.",
+        default="SAT",
+    )
+    suspectMaskName = pexConfig.Field(
+        dtype=str,
+        doc="Name of mask plane to use for suspect pixels.",
+        default="SUSPECT",
+    )
+
+    # Crosstalk.
+    doCrosstalk = pexConfig.Field(
+        dtype=bool,
+        doc="Apply intra-CCD crosstalk correction?",
+        default=False,
+    )
+    doCrosstalkBeforeAssemble = pexConfig.Field(
+        dtype=bool,
+        doc="Apply crosstalk correction before CCD assembly?",
+        default=False,
+    )
+    crosstalk = pexConfig.ConfigurableField(
+        target=CrosstalkTask,
+        doc="Intra-CCD crosstalk correction.",
+    )
+
+    # Masking options.
+    doDefect = pexConfig.Field(
+        dtype=bool,
+        doc="Apply correction for CCD defects, e.g. hot pixels?",
+        default=True,
+    )
+    doNanMasking = pexConfig.Field(
+        dtype=bool,
+        doc="Mask non-finite (NAN, inf) pixels.",
+        default=True,
+    )
+    doWidenSaturationTrails = pexConfig.Field(
+        dtype=bool,
+        doc="Widen bleed trails based on their width.",
+        default=True,
+    )
+    doCameraSpecificMasking = pexConfig.Field(
+        dtype=bool,
+        doc="Mask camera-specific bad regions?",
+        default=False,
```

Comment:
What does this config do?

Reply:
Here as well, there are probably historical reasons for needing to do camera-specific masking, happy to get rid of it if we are doing such masking in another way.

Reply:
I think we should remove this from the LSST task, it's not relevant.